ansible/roles/oracle-db-monitoring/tasks/install-oracle-db-monitoring.yml

```diff
@@ -1,7 +1,11 @@
-- name: Get monitoring tags as a list
+- name: Get oracle database sids for monitoring as a list from the oracle-sids tag
   set_fact:
     oracle_monitoring_list: "{{ ec2.tags['oracle-sids'].split() | default([]) }}"
```

Comment:
You might need a conditional on this as well, e.g. if oracle-sids is not defined

Reply:
done